### PR TITLE
Adds the process for deprecating meta objects

### PR DESCRIPTION
**Pull request by codeZeilen**

### CODE REVIEW

File: _pages/development_process.md [hunk 1]
```diff
@@ -55,6 +55,23 @@ In case there are no new commits in the trunk repository, core developers can al
  5. Update the trunk update config with the latest version of packages - and REMOVE the package from the config that should no longer be in trunk. Save this update.
  6. Test it with a new image to verify everything worked correctly.
 
+## How to Deprecate a Class or Method
+When a single meta object is removed from the image we want to make sure that projects relying on it can still be loaded in newer Squeak versions. Therefore, these methods and classes are saved in special packages. The packages are named using the following pattern: "*NNDeprecated-OriginalPackageName*" (NN being the two digit version number). So, if we want to deprecate meta objects from the "*Kernel*" package which were present up to Squeak 5.1 we would put them into the package: "*51Deprecated-Kernel*".
```
Comment (krono) [inline _pages/development_process.md]: "which were present" is confusing. Rather "which where part of the base system" or so

**Comment by fniephaus** [comment]
@codeZeilen *ping*

**Comment by codeZeilen** [comment]
will do on monday :)

**Comment by codeZeilen** [comment]
Ok I am happy with it now :)

**Comment by fniephaus** [comment]
Spell check passes, merging it in...

**Pull request merged**